python_modules/libraries/dagster-dbt/dagster_dbt/include/scaffold/definitions.py.jinja

```diff
@@ -7,6 +7,14 @@ from dagster_dbt import DbtCli, dbt_assets
 dbt_project_dir = Path(__file__).parent.joinpath({{ dbt_project_dir_relative_path_parts | join(', ')}})
 dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
 
+dbt = DbtCli(project_dir=os.fspath(dbt_project_dir))
+
+if os.getenv("DAGSTER_DBT_BUILD_PROJECT"):
+    dbt_parse_invocation = dbt.cli(["parse"], manifest={})
```

Comment:
We theoretically don't need the manifest argument here, right? I filed https://github.com/dagster-io/dagster/issues/15233 to track.